src/CounterInterceptor/MyChildWorkflow.workflow.cs

```diff
@@ -0,0 +1,21 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Workflows;
+
+[Workflow]
+public class MyChildWorkflow
+{
+    private readonly ActivityOptions activityOptions = new()
+    {
+        StartToCloseTimeout = TimeSpan.FromSeconds(10),
+    };
+
+    [WorkflowRun]
+    public async Task<string> ExecChildAsync(string name, string title)
```

Comment:
```suggestion
    public async Task<string> RunAsync(string name, string title)
```

Not required obviously, but I think calling all run calls `RunAsync` is clearest

Reply:
Yeah I agree. Fixed.